webpack/assets/javascripts/react_app/components/HostDetails/index.js

```diff
@@ -54,9 +54,9 @@ const HostDetails = ({ match, location: { hash } }) => {
     registerCoreTabs();
   }, []);
 
-  useEffect(() => {
-    if (hash) setActiveTab(hash.slice(1));
-  }, [hash]);
+  const capitalize = tabName => `${tabName[0].toUpperCase()}${tabName.slice(1)}`
```

Comment:
I do not like fiddling with capitalization, it would be good to have the ability to register multiple values for a tab - tab header label and a path segment.